packages/plugins/GoPlusSecurity/src/locales/languages.ts

```diff
@@ -25,7 +25,7 @@ if (import.meta.webpackHot) {
             globalThis.dispatchEvent?.(
                 new CustomEvent('MASK_I18N_HMR', {
                     detail: [
-                        'io.mask.debugger',
+                        'io.gopluslabs.security',
```

Comment:
DO NOT change this file manually. You need to change `/.i18n-codegen.json` and re-run `npx gulp sync-languages`